### DISCUSSION TITLE
Enhancement - add pac webresource push

### DISCUSSION THREAD

**Discussion by dmcweeney** [microsoft/powerplatform-build-tools#770] [Ideas]
Hi,

This is an enhancement suggestion I originally added to the powerplatform-vscode repo back in 2022 - [Enhancement - add pac webresource push](https://github.com/microsoft/powerplatform-vscode/discussions/661) - but no response after 16 months.

As suggested in a few other discussions, making the pac and powerplatform-cli-wrapper repos public would be a big step forward.

**Comment by kalluu-ck** [comment]
Hello, any updates on this enhancement request please?? :D

**Comment by dmcweeney** [comment]
@kalluu-ck 

This [tool](https://github.com/neronotte/Greg.Xrm.Command/wiki/pacx-webresources-push) may help...
